### ISSUE TITLE
Certificate expired 2017-04-24

### ISSUE THREAD

**Issue by davicus1** [jfelchner/ruby-progressbar#135]
## Current Behavior
I attempted to install the gem with HighSecurity, and I get the following error:
```
ERROR:  While executing gem ... (Gem::Security::Exception)
    certificate /CN=accounts_rubygems/DC=thekompanee/DC=com not valid after 2017-04-24 02:51:23 UTC
```

### Steps to Reproduce 
```
gem cert --add <(curl -Ls https://raw.github.com/jfelchner/ruby-progressbar/master/certs/jfelchner.pem)
sudo gem install ruby-progressbar -P HighSecurity
```

**Comment by jfelchner** [comment]
Thank you!!!

**Comment by jfelchner** [comment]
All, https://github.com/jfelchner/ruby-progressbar/commit/d4657abde6e477767e9156053e7d0065cab3b0ef fixes this issue and is released as of [1.8.3](https://github.com/jfelchner/ruby-progressbar/tree/releases/v1.8.3).  @davicus1 thank you SO MUCH for bringing this to my attention. 😀
